teleop_twist_keyboard.py

```diff
@@ -19,18 +19,30 @@
 e/c : increase/decrease only angular speed by 10%
 anything else : stop
 
+For Horonomic base, holding down Shift key
+
+For Drones, use t/b to control elevation
```

Comment:
I'd like to propose:

> For Holonomic mode (strafing), hold down the shift key:
> ```
 U  I  O
 J  K  L
 M  <  >
```
> 
> For vertical movement, use `t` for up and `b` for down, at the same speed.
>
> Holonomic mode and vertical mode both use the linear speed setting.

For properly implementing Holonomic mode, this also needs bindings for `I` and `<`